assets/scss/decisions.scss

```diff
@@ -0,0 +1,115 @@
+.decision-title {
+    margin-bottom: govuk-spacing(7);
+}
+
+.decision-details {
+    @include govuk-font(19);
+    font-weight: bold;
+
+    .decision-published-date {
+        margin-bottom: 20px;
+    }
+}
+
+.decision-summary__link {
+    margin-top: govuk-spacing(6);
+}
+
+#primary .decision-summary {
+    &__image {
+        width: 198px;
+        padding-bottom: 2px;
+
+        a {
+            display: block;
+
+            &:focus img {
+                outline: none;
+            }
+        }
+
+        img {
+            max-width: 100%;
+            max-height: 264px;
+            outline: 1px solid $govuk-border-colour;
+            mix-blend-mode: multiply;
+        }
+    }
+
+    &__icon {
+
+        svg,
+        img {
+            position: relative;
+            top: 3px;
+            padding-right: 0.5em;
+        }
+
+        svg {
+            height: 1.2em;
+        }
+
+        img {
+            height: 1em;
+            width: auto;
+        }
+    }
+
+    .decision-summary-text p {
+        font-weight: bold;
+        margin-bottom: 20px;
+    }
+}
```

Comment:
I don't think any of this is needed.  I can't find where it would be applied.

Reply:
Yes, the summary stuff could go. I left some summary code in the listing page with a probably misguided belief that they would be wanted the summary stuff added in. I noticed all our custom post types have their own styles even for page elements that do the same thing. We could probably refactor all of these cpt css classes to use mostly the same ones.

Reply:
Yes, that's probably a good idea.  You could leave this in then and let it be swept away in that refactor work.